### PR TITLE
Add startup scripts

### PR DESCRIPTION
**Pull request by rugk**
Taken from Etherpad and modified for OneTimeSecret.

### CODE REVIEW

File: bin/run.sh [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+for ARG in $*
+do
+  if [ "$ARG" = "--root" ]; then
+    ignoreRoot=1
+  fi
+done
+
+#Stop the script if its started as root
+if [ "$(id -u)" -eq 0 ] && [ $ignoreRoot -eq 0 ]; then
+   echo "You shouldn't start OneTimePSecret as root!"
```
Comment (kevinkupski) [inline bin/run.sh]: There is still a typo ;-)

**Comment by delano** [comment]
Can you move the license to `licenses/etherpad.txt` and add a note to the readme that the project includes their-party software which links to both [etherpad](https://github.com/ether/etherpad-lite) and that copy of the license.

**Comment by rugk** [comment]
Done.

**Comment by rugk** [comment]
Anything else to do?

**Comment by fraenki** [comment]
@rugk: One question, there is no PID file written (except for redis). How do we properly stop the service then? I think it would be nice to be able to start _and_ stop the service.

**Comment by rugk** [comment]
This can be done in the [service file](https://github.com/ether/etherpad-lite/wiki/How-to-deploy-Etherpad-Lite-as-a-service).

**Comment by delano** [comment]
Thanks for this. Apologies for the delay in merging.

**Pull request merged**